include/seqan3/io/record.hpp

```diff
@@ -64,7 +66,7 @@ enum class field
     seq,            //!< The "sequence", usually a range of nucleotides or amino acids.
     id,             //!< The identifier, usually a string.
     qual,           //!< The qualities, usually in Phred score notation.
-    seq_qual,       //!< Sequence and qualities combined in one range.
+    seq_qual SEQAN3_DEPRECATED_310, //!< [DEPRECATED] Sequence and qualities combined in one range.
```

Comment:
I'm wondering whether

```suggestion
    _seq_qual_deprecated, //!< [DEPRECATED] Sequence and qualities combined in one range.
    seq_qual SEQAN3_DEPRECATED_310 = _seq_qual_deprecated, //!< [DEPRECATED] Sequence and qualities combined in one range.
```